src/main/java/com/spikes2212/command/genericsubsystem/smartmotorcontrollersubsystem/CTRESmartMotorControllerGenericSubsystem.java

```diff
@@ -109,7 +110,8 @@ public void pidSet(UnifiedControlMode controlMode, double setpoint, PIDSettings
                        FeedForwardSettings feedForwardSettings, TrapezoidProfileSettings trapezoidProfileSettings) {
         configPIDF(pidSettings, feedForwardSettings);
         configureTrapezoid(trapezoidProfileSettings);
-        master.set(controlMode.getCTREControlMode(), setpoint);
+        master.set(controlMode.getCTREControlMode(), setpoint, DemandType.ArbitraryFeedForward,
+                feedForwardSettings.getkS() * Math.signum(setpoint) + feedForwardSettings.getkG());
```

Comment:
what if my position is `-600` and my setpoint is `-300`, would the `Math.signum` work then?